integration/ts-compat/import-all-entry-points.bzl

```diff
@@ -0,0 +1,44 @@
+load("//src/cdk:config.bzl", "CDK_ENTRYPOINTS")
+load("//src/cdk-experimental:config.bzl", "CDK_EXPERIMENTAL_ENTRYPOINTS")
+load("//src/material:config.bzl", "MATERIAL_ENTRYPOINTS", "MATERIAL_TESTING_ENTRYPOINTS")
+load("//src/material-experimental:config.bzl", "MATERIAL_EXPERIMENTAL_ENTRYPOINTS", "MATERIAL_EXPERIMENTAL_TESTING_ENTRYPOINTS")
+
+"""Converts the given string to an identifier."""
+
+def convert_to_identifier(name):
+    return name.replace("/", "_").replace("-", "_")
+
+"""Creates imports and exports for the given entry-point and package."""
+
+def create_import_export(entry_point, pkg_name):
+    identifier = "%s_%s" % (convert_to_identifier(pkg_name), convert_to_identifier(entry_point))
+    return """
+    import * as {0} from "@angular/{1}/{2}";
+    export {{ {0} }};
+  """.format(identifier, pkg_name, entry_point)
+
+"""
+  Creates a file that imports all entry-points as namespace. The namespaces will be
+  re-exported. This ensures that all entry-points can successfully compile.
+"""
+
+def generate_import_all_entry_points_file():
```

Comment:
Is there a way we could consolidate on a single, canonical list of packages for the repo? Right now I think there's a handful of places that need to be updated when we add a new package. Either that, or just documenting all the places that need to be updated

Reply:
💯 Definitely agreed. I've been trying to reduce this type of duplication (as seen with entry-points in the system config etc.). 

I'll work on a follow-up that gives us a better list of release packages. The problem always comes from the fact that we can't easily access these variables outside of Bazel (e.g. in the release scripts).